.infrastructure/55_cognito.tf

```diff
@@ -330,3 +330,134 @@ resource "aws_cognito_user_pool_domain" "devportal" {
   certificate_arn = aws_acm_certificate.auth.arn
   user_pool_id    = aws_cognito_user_pool.devportal.id
 }
+
+resource "aws_cognito_identity_pool" "devportal" {
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  identity_pool_name               = "devportal-identity"
+  allow_unauthenticated_identities = false
+  allow_classic_flow               = false
+
+  cognito_identity_providers {
+    client_id               = aws_cognito_user_pool_client.devportal_website.id
+    provider_name           = aws_cognito_user_pool.devportal.endpoint
+    server_side_token_check = false
+  }
+
+}
+
+resource "aws_iam_role" "devportal_authenticated_user" {
+  name        = "DevPortalAuthenticatedUser"
+  description = "The role assumed by the authenticated devportal users"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow"
+        Action = "sts:AssumeRoleWithWebIdentity",
+        Principal = {
+          "Federated" : "cognito-identity.amazonaws.com"
+        }
+        Condition = {
+          StringEquals = {
+            "cognito-identity.amazonaws.com:aud" : aws_cognito_identity_pool.devportal.id
+          }
+          "ForAnyValue:StringLike" = {
+            "cognito-identity.amazonaws.com:amr" : "authenticated"
+          }
+        }
+      }
+    ]
+  })
```

Comment:
Maybe it is possible to avoid duplication with the same `assume_role_policy` of the `devportal_authenticated_host_user` resource

Reply:
Done with commit [81507d5](https://github.com/pagopa/developer-portal/pull/576/commits/81507d5d3c5a224ebe32188c511528a0553a9cc3)